package.json

```diff
@@ -14,28 +14,5 @@
   "repository": {
     "type": "git",
     "url": "git@github.com:sirensolutions/kibi-radar-chart-plugin.git"
-  },
-  "scripts": {
```

Comment:
Please leave the script and devDependencies section 
They are used by siren team